backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationService.java

```diff
@@ -102,7 +110,7 @@ public void assertFacilityNameAvailable(String testingFacilityName) {
     }
 
     public Facility getFacilityInCurrentOrg(UUID facilityId) {
-        Organization org = getCurrentOrganization();
+        Organization org = this.getCurrentOrganization();
```

Comment:
?

Reply:
even _i_ don't remember adding that. oops.